CONTRIBUTING.md

```diff
@@ -66,17 +66,16 @@ _Setting up a running Ruby environment is outside the scope of this document._
 
 This repository contains multiple Ruby gems:
 
- *  `opentelemetry-api` located in the `api` directory
- *  `opentelemetry-sdk` located in the `sdk` directory
  *  Various instrumentation gems located in subdirectories of `instrumentation`
- *  Various exporter gems located in subdirectories of `exporter`
+ *  `opentelemetry-propagator-xray` located in the `propagator/xray` directory
+ *  `opentelemetry-propagator-ottrace` located in the `propagator/ottrace` directory
```

Comment:
I noticed that these propagator gems were not listed here in the doc, are they worth mentioning like this?

Reply:
Sure! If others would find it helpful.